care/facility/migrations/0081_auto_20200409_1201.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.11 on 2020-04-09 06:31
+
+from django.db import migrations, models
+import fernet_fields.fields
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'fernet_fields.fields'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5332931)